pkg/manifests/external_dns.go

```diff
@@ -297,3 +383,58 @@ func newExternalDNSDeployment(conf *config.Config, externalDnsConfig *ExternalDn
 		},
 	}
 }
+
+func getServiceAccount(externalDnsConfig *ExternalDnsConfig) string {
```

Comment:
method on externalDnsConfig instead?

Reply:
resolving since I moved this to constructor